automation/test-linux.sh

```diff
@@ -63,43 +77,60 @@ fi
 
 delete_vm(){
   vm_name=$1
-  local template_name=$2
+
+  if [ "${KUBE_CMD}" == "oc" ]; then
```

Comment:
can you please make this variable `template_option` as function param? Because on line 114 you have identical if statement

Reply:
Done